### PR TITLE
Update OmniSharp version to 1.37.9

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by JoeRobich** [comment]
Seeing some completion test failures. @333fred Can you take a look?

**Comment by 333fred** [comment]
https://github.com/JoeRobich/omnisharp-vscode/pull/2 should fix the tests.

**Comment by JoeRobich** [comment]
thanks @333fred!

**Comment by 333fred** [comment]
Oh no, I think the failure here is a newline difference on windows vs linux...

File: test/integrationTests/completionProvider.integration.test.ts [hunk 1]
```diff
@@ -42,30 +42,39 @@ suite(`${OmniSharpCompletionProvider.name}: Returns the completion items`, () =>
...
-            expect(i.additionalTextEdits[0].range.end.character).equals(16);
+            if (i.additionalTextEdits) {
+                sawAdditionalTextEdits = true;
+                expect(i.additionalTextEdits).to.be.array();
+                expect(i.additionalTextEdits.length).to.equal(1);
+                expect(i.additionalTextEdits[0].newText).to.equal("using singleCsproj2;\n");
+                expect(i.additionalTextEdits[0].range.start.line).to.equal(1);
+                expect(i.additionalTextEdits[0].range.start.character).to.equal(0);
+                expect(i.additionalTextEdits[0].range.end.line).to.equal(1);
+                expect(i.additionalTextEdits[0].range.end.character).to.equal(1);
```
Comment (333fred) [inline test/integrationTests/completionProvider.integration.test.ts:70]:
```suggestion
                // Can be either 0 or 1, depending on the platform this test is run on
                expect(i.additionalTextEdits[0].range.end.character).to.be.lessThanOrEqual(1).and.greaterThanOrEqual(0);
```

**Comment by 333fred** [comment]
Alright, that failure at least wasn't the completion changes 😅

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,22 @@
   * As a workaround, make an edit within the file before using Rename Symbol.
 
 ## 1.23.12 (Not yet released)
+* Support async completion (PR: [#4116](https://www.github.com/OmniSharp/omnisharp-vscode/pull/4116))
```
Comment (333fred) [inline CHANGELOG.md:15]:
```suggestion
* Support experimental async completion (PR: [#4116](https://www.github.com/OmniSharp/omnisharp-vscode/pull/4116))
```

File: README.md [hunk 3]
```diff
@@ -27,6 +27,28 @@ The .NET 5 SDK requires version 16.8 of MSBuild.
 For Windows users who have Visual Studio installed, this means you will need to be on the latest Visual Studio 16.8 Preview.
 For MacOS and Linux users who have Mono installed, this means you will need to set `omnisharp.useGlobalMono` to `never` until a version of Mono ships with MSBuild 16.8.
 
+## What's new in 1.23.12
+- Support async completion (PR: [#4116](https://www.github.com/OmniSharp/omnisharp-vscode/pull/4116))
```
Comment (333fred) [inline README.md:31]:
```suggestion
- Support experimental async completion (PR: [#4116](https://www.github.com/OmniSharp/omnisharp-vscode/pull/4116))
```

**Review by 333fred — APPROVED** [review]
Couple of small clarifications, otherwise LGTM.

**Pull request merged**